src/main/java/mat/client/shared/MeasureLibraryGridToolbar.java

```diff
@@ -90,6 +103,23 @@ public void updateOnSelectionChanged(Collection<ManageMeasureSearchModel.Result>
 
         historyButton.setEnabled(true);
 
+        //Run FHIR validation Button
```

Comment:
1. **Responsibility:** I think it's not the responsibility of a toolbar to do this business logic.

See how selectedItem.isSharable() or selectedItem.isExportable() are implemented. In a similar way you can introuce selectedItem.isValidatable()

2. RegExp parsing is more expensive and verbose then if you parse versions as BigDecimal.